week-03/main.js

```diff
@@ -3,34 +3,56 @@
 // Given the following array
 var list = [1, 2, 3, 4, 5];
 // Remove one from the beginning of the array and add it to the end
-
-
+list.shift();
+list.push(1);
 // # Objects
 
 // Create an object with key value pairs for
 // `name` - a string,
 // `isAlive` - a boolean,
 // and `hobbies` an array.
 // Assign the object to the variable `user`
+var user = {name: 'margie', isAlive: true, hobbies:[]};
 
 
 
-// Add 'golf' to the `user`'s `hobby` array.
 
+// Add 'golf' to the `user`'s `hobby` array.
+user.hobbies = 'golf';
 
 // # Functions
 
 // Define a function `sayHello` that takes a name string and returns "Hello" plus that name and an exclamation point.
 // Example:
 
 // sayHello('Mer') // "Hello Mer!"
+var sayHello = function(name) {
+    return 'Hello ' + name + '!';
+}sayHello('Mer');
```

Comment:
yes, this works much better :)
var sayHello = function(name) {
    return 'Hello ' + name + '!';
}
sayHello('Mer');